.dockerignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
This is the most important thing, but it might be worth adding any caches (e.g. `**/__pycache__`) as well